src/GridapGeosciences.jl

```diff
@@ -28,7 +30,11 @@ module GridapGeosciences
   export Eₖ, Eₚ, Eₜ
   export compute_kin_to_pot!, compute_pot_to_kin!, compute_total_mass!
   export dump_diagnostics_shallow_water!
+  export compute_mass_flux!, compute_bernoulli_potential!, compute_potential_vorticity!
   export shallow_water_explicit_time_step!
-  export shallow_water_time_stepper
+  export shallow_water_explicit_time_stepper
+  export shallow_water_rosenbrock_time_step!
```

Comment:
Do we need to export `shallow_water_rosenbrock_time_step!` ? If yes, why?

The same applies to `shallow_water_explicit_time_step!

Reply:
what about the rest of symbols? Which ones need to be actually exported?

Reply:
No, I don't think we do, this was a legacy from before the time steppers functions were implemented. I'll fix and look for other symbols we don't need to export...